src/Resolver.js

```diff
@@ -147,7 +147,7 @@ class Resolver {
 
       default:
         // Module
-        return path.normalize(filename);
+        return path.normalize(filename).replace(/\\/g, '/');
```

Comment:
Not sure why this even normalizes the filename?

This wouldn't do anything if it really is a node_module right?

`some-module` => `some-module`
`some/sub/module/file.js` => `some/sub/module/file.js`

I'll remove it for performance reasons. (and it doesn't fail any tests, it even fixes one on windows)